CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace CryptoTechReminderSystem.Gateway
+{
+    public class HarvestGateway
+    {
+        private readonly HttpClient _client;
+        private string _token;
+
+        public HarvestGateway(string address, string token)
+        {
+            _client = new HttpClient { BaseAddress = new Uri(address) };
+            _token = token;
+        }
+
+        public string Retrieve()
```

Comment:
I'd expect a gateway to return a domain object here, rather than the raw JSON. Otherwise the use case & test assertions will be coupled to the JSON that is only specific to Harvest

Reply:
We have also utilised a domain object for it to be returned in the gateway in bcb8cc131808ddebd5be51d4f1ef6e32fb3bd26a.